src/mangle.c

```diff
@@ -901,3 +901,26 @@ void mangleToBuffer(Expression *e, OutBuffer *buf)
     Mangler v(buf);
     e->accept(&v);
 }
+
+void mangleToFuncSignature(OutBuffer *buf, FuncDeclaration *fd)
```

Comment:
I don't understand why there is a same-only-different type signature routine. Why should this produce different results than tf->deco? And if it does produce something different, isn't something terribly wrong somewhere?